packages/common/utils/src/reverseKeyValue.ts

```diff
@@ -1,4 +1,14 @@
 /** @tossdocs-ignore */
+
 export function reverseKeyValue<KeyType extends string, ValueType extends string>(obj: Record<KeyType, ValueType>) {
-  return Object.fromEntries(Object.entries(obj).map(([key, value]) => [value, key])) as Record<ValueType, KeyType>;
+  const reversedObj: Record<ValueType, KeyType | KeyType[]> = {} as Record<ValueType, KeyType | KeyType[]>;
```

Comment:
```suggestion
  const reversedObj = {} as Record<ValueType, KeyType | KeyType[]>;
```

I think this type define is not required, cause `as` assertion is used.